tests/tests_unit/rules/test_analysis.py

```diff
@@ -68,7 +68,7 @@ def test_directly_referred_classes(self, olav_rules: InformationRules) -> None:
         assert len(InformationArchitectRulesAnalysis(olav_rules).directly_referred_classes) == 3
 
     def test_inherited_referred_classes(self, olav_rules: InformationRules) -> None:
-        assert len(InformationArchitectRulesAnalysis(olav_rules).inherited_referred_classes) == 2
+        assert len(InformationArchitectRulesAnalysis(olav_rules).inherited_referred_classes) == 1
 
     def test_referred_classes(self, olav_rules: InformationRules) -> None:
-        assert len(InformationArchitectRulesAnalysis(olav_rules).referred_classes) == 5
+        assert len(InformationArchitectRulesAnalysis(olav_rules).referred_classes) == 4
```

Comment:
New Classes
![image](https://github.com/cognitedata/neat/assets/60234212/89ece107-c01e-4fe3-bc57-6dad047c0320)

Reply:
Not sure fi this is the correct behavior:
![image](https://github.com/cognitedata/neat/assets/60234212/dbd5bdf0-6e79-413d-9d94-4d7ac5923176)